manual/dmc.tex

```diff
@@ -37,7 +37,7 @@ \section{Diffusion Monte Carlo}
    &   \texttt{reconfiguration     } &  string  & yes/pure/other & no   & fixed population technique  \\
    &   \texttt{branchInterval      } &  integer  & $\ge 0$ & 1   & branching interval \\
    &   \texttt{substeps            } &  integer  & $\ge 0$ & 1   & branching interval \\
-   &   \texttt{nonlocalmoves       } &  string  & yes/other & no   & run with tmoves  \\
+   &   \texttt{nonlocalmoves       } &  string  & yes/v0/v1/other & no   & run with tmoves  \\
```

Comment:
It appears that 'other' is longer an option (I assume here that 'other' means 'a string other than yes or no or v0 or v1', and not the exact string 'other').   The new code gives an error for a string that is not yes,v0,v1, or no.

Reply:
Based on the manual description I am happy with the new error code and deprecating "other". I don't believe that it was widely used and find the vagueness dangerous. In my experience the nonlocalmoves parameter is only specified to enable it, "yes", so the number of input file affected by this change will be tiny if not zero. Reasonable?

Reply:
''other'' means anything not ''yes''. After the change, the code will complain. We only used ''yes'' to access the Tmoves. Sometimes when I need to switch it off without deleting the input line, I put ''no'' as a natural response. So the number of input files affected should be zero or it is a user mistake.